frontend/src/shared/helpers.tsx

```diff
@@ -177,37 +177,30 @@ export const PatientsListColumns: ColDef[] = [
   {
     field: "cmoPatientId",
     headerName: "CMO Patient ID",
-    sortable: false,
   },
   {
     field: "dmpPatientId",
     headerName: "DMP Patient ID",
-    sortable: false,
   },
   {
     field: "totalSampleCount",
     headerName: "# Samples",
-    sortable: false,
   },
   {
     field: "cmoSampleIds",
     headerName: "Sample IDs",
-    sortable: false,
   },
   {
     field: "consentPartA",
     headerName: "12-245 Part A",
-    sortable: false,
   },
   {
     field: "consentPartC",
     headerName: "12-245 Part C",
-    sortable: false,
   },
   {
     field: "smilePatientId",
     headerName: "SMILE Patient ID",
-    hide: true,
```

Comment:
this should still be hidden by default. Was this causing any issues or was this a mistake?

Reply:
This was a mistake. Thanks for catching!